### PR TITLE
IECISA new machine

### PR DESCRIPTION
**Pull request by onmax**
# Hi! Thank you for submitting a Pull Request!

## If this is not for an enode you can ignore this and delete it.

ENODE: enode://0f92d0367ecc129dd5d3bc7d0cf271a6c4a8ac0408ba3262dc2434d73e7b16651d69b8942f3c59943e47202f79cfe879a2c73b05838006773212087aa7c57b45@34.91.211.67:21000?discport=0
--

### CODE REVIEW

**Comment by onmax** [comment]
This PR:
-  removes one of our machines 
- edits the information of other machine

**Review by VictorNS69 — CHANGES_REQUESTED** [review]
I think you need to change 2 more files as the installation guide says.
image
Guide link: https://medium.com/babel-go2chain/setting-in-motion-a-regular-node-in-the-telsius-network-of-alastria-c2d67b8369c7

The changes you need to make are:
1. https://github.com/onmax/alastria-node/blob/testnet2/data/regular-nodes.json#L34: **Change the old enode for the new one**
2. https://github.com/onmax/alastria-node/blob/testnet2/data/regular-nodes.json#L135: **Remove the deleted enode**
3. https://github.com/onmax/alastria-node/blob/testnet2/data/constellation-nodes.json#L30: **Remove this IP from the list**
4. https://github.com/onmax/alastria-node/blob/testnet2/data/constellation-nodes.json#L117: **Remove this IP from the list**

I think those are all the changes you have to make.

File: DIRECTORY_REGULAR.md [hunk 1]
```diff
@@ -32,7 +32,7 @@
...
 | SIEMENS | AWS (2C/4Gb/512GiB) | 71VEFpNN8qJOXpidQA/JfWCWUoh+078Pq3E5elKJCSI= | enode://b784aeb62629bb7b1acdab5ca3aaac74505c70d66b5e5a4a44224c93c51e2929046d6ca242c443973caddf59173c7db37707608b3788235cf51ab4d086b80115@18.196.97.221:21000?discport=0 |
 | SM_SERVICES | IBM Cloud (2C/2Gb/50Gb)| J5Cv0/XiIavBbq4t09X3hFDxDnxCg7yBtBB7uO03bSU= | enode://c9394766eb2816be19a8a5c06f6c251b55b273a54f9b1cf3440c7984506d801910216beb8772eca15496005c4eea7d7344645778d18fb7b17c05077046785bdc@159.122.23.130:21000?discport=0 |
-| IECISA | KIO (2C/4Gb/100Gb) | sfbk8BqQnsz+eztwhoL7cYwBGSAwg+8WRrLCjXOP+Fk= | enode://97cf63eb56893df0e5626833e20a490dd88e451ee6d541ff8aa8629d6377ffa800b6067d5777ac29c7bac6ee0863eec40df05b65f6c9469d3e1f42c694cfe717@188.210.217.181:21000?discport=0 |
+| Inetum | Google Cloud (2C/4Gb/100Gb) | sfbk8BqQnsz+eztwhoL7cYwBGSAwg+8WRrLCjXOP+Fk= | enode://0f92d0367ecc129dd5d3bc7d0cf271a6c4a8ac0408ba3262dc2434d73e7b16651d69b8942f3c59943e47202f79cfe879a2c73b05838006773212087aa7c57b45@34.91.211.67:21000?discport=0 |
```
Comment (VictorNS69) [inline DIRECTORY_REGULAR.md:35]:
```suggestion
| Inetum | Google Cloud (2C/4Gb/100Gb) |  | enode://0f92d0367ecc129dd5d3bc7d0cf271a6c4a8ac0408ba3262dc2434d73e7b16651d69b8942f3c59943e47202f79cfe879a2c73b05838006773212087aa7c57b45@34.91.211.67:21000?discport=0 |
```
If Constellation is not enable, please remove that. Also, if Constellation is enabled, how it has the same key?

**Review by VictorNS69 — CHANGES_REQUESTED** [review]

File: DIRECTORY_REGULAR.md (hunk 1 above)
Comment (onmax) [reply DIRECTORY_REGULAR.md:35]: Published the changes

**Review by VictorNS69 — APPROVED** [review]
Now sounds good to me. Let's wait for @alejandroalffer's approval.

**Pull request merged**